test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change_with_blacklist.test

```diff
@@ -0,0 +1,492 @@
+Fiware-Servicepath: /
+Content-Length: 131
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "C": {
+                "metadata": {}, 
+                "type": "Integer", 
+                "value": 32
+            },			
+            "id": "Room1", 
+            "type": "Room"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+10. Sleep 5 secs, before next update.
+=====================================
+
+
+11. PATCH /v2/entities/Room1/attrs, update the value of C and D attribute.
+==========================================================================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+12. Dump and reset accumulator, see notification.
+=================================================
+POST http://localhost:REGEX(\d+)/notify
+Fiware-Servicepath: /
+Content-Length: 131
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "C": {
+                "metadata": {}, 
+                "type": "Integer", 
+                "value": 34
+            }, 
+            "id": "Room1", 
+            "type": "Room"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+13. Sleep 5 secs, before next update.
+=====================================
+
+
+14. PATCH /v2/entities/Room1/attrs, update the value of A, B, C and D attribute.
+================================================================================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+15. Dump and reset accumulator, see notification.
+=================================================
+POST http://localhost:REGEX(\d+)/notify
+Fiware-Servicepath: /
+Content-Length: 179
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: normalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A": {
+                "metadata": {}, 
+                "type": "Integer", 
+                "value": 40
+            },
+            "C": {
+                "metadata": {}, 
+                "type": "Integer", 
+                "value": 20
+            },			
+            "id": "Room1", 
+            "type": "Room"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+
+
+--TEARDOWN--
+brokerStop CB
+accumulatorStop $LISTENER_PORT
+dbDrop CB
```

Comment:
I'd suggest to add an additional .test to check the basic subscription CRU with regards to the new field. For instance, these steps:

* Create a subscription with onlyChanged: true
* Get (GET) subscription to check onlyChanged: true is rendered
* Modify (PATCH) subscription to change onlyChanged to false
* Get (GET) subscription to check onlyChanged: false is rendered

Reply:
fixed in f9887f1